docs/setup/import.md

```diff
@@ -200,7 +200,7 @@ lakefs import --with-merge lakefs://example-repo -m s3://example-bucket/path/to/
 
 1. You can download the `lakefs` binary from [here](https://github.com/treeverse/lakeFS/releases). Make sure you choose one compatible with your installation of lakeFS.
 
-1. Use a configuration file like the one used to start your lakeFS installation. This will be used to access your database. An example can be found [here](../reference/configuration.html#example-aws-deployment).
+1. Use a configuration file like the one used to start your lakeFS Installation. This will be used to access your database. An example can be found [here](../reference/configuration.html#example-aws-deployment).
```

Comment:
Why?

Reply:
I've seen installation written with a capital letter somewhere in the docs and assume this is how it should be, some kind of lakeFS standard. If not, then there's definitely no reason to capitalize it

Reply:
I think it was a typo